modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -1805,6 +1876,18 @@
           end
         end
 
+        context 'when the activeDutyBeginDate are null' do
+          it 'responds with a 422', skip: 'needs 2nd opinion' do
+            mock_ccg(scopes) do |auth_header|
```

Comment:
@FonzMP Would you mind pulling this down and running this test? Take a look at the response.body, do you get a PG error regarding a null claim_id?

Reply:
I am not FonzMP, but I did run this test locally and got a PG::NotNullViolation:

{\"errors\":[{\"title\":\"Internal server error\",\"detail\":\"Internal server error\",\"code\":\"500\",\"status\":\"500\",\"meta\":{\"exception\":\"PG::NotNullViolation: ERROR:  null value in column \\\"claim_id\\\" violates not-null constraint\\n